### PR TITLE
Add travis integration, vendoring

### PR DESCRIPTION
**Pull request by Raffo**
Just a small step to at least have the tests being run. I also added vendoring as well. 

I can add `goveralls` as well if @mikkeloscar thinkks this is welcome.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- tip
+
+before_install:
+- go get github.com/Masterminds/glide
+- go get github.com/alecthomas/gometalinter
+- go get -u github.com/go-swagger/go-swagger/cmd/swagger
+
+install:
+- glide install --strip-vendor
+- gometalinter --install
```
Comment (mikkeloscar) [inline .travis.yml]: I don't think `gometalinter --install` is needed as it's not required by `make test`.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+language: go
+
+go:
+- 1.9
+- tip
+
+before_install:
+- go get github.com/Masterminds/glide
+- go get github.com/alecthomas/gometalinter
+- go get -u github.com/go-swagger/go-swagger/cmd/swagger
```
Comment (mikkeloscar) [inline .travis.yml]: Is gometalinter and go-swagger needed? I don't think they are required by `make test`.

Comment (Raffo) [reply .travis.yml]: I wanted to add linting as well and make it already "ready" for the future. I can remove it now. 

I don't know if go-swagger is not needed, easier to remove it and see.

File: .travis.yml (hunk 1 above)
Comment (Raffo) [reply .travis.yml]: as above

File: .travis.yml (hunk 2 above)
Comment (mikkeloscar) [reply .travis.yml]: Can you remove `go-swagger` so we can see? :) Any dependencies should be included with `glide install`.

Let's keep gometalinter!

**Comment by Raffo** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**